include/nccl_ofi.h

```diff
@@ -42,6 +42,9 @@ extern "C" {
  */
 #define OFI_HIGHEST_TAG_BIT		(0x1UL << 63)
 
+
+#define NCCL_OFI_EXPORT_SYMBOL __attribute__((visibility("default")))
```

Comment:
This should be in nccl_ofi_config_bottom.h, because not every file includes nccl_ofi.h.

Reply:
moved it into the second config.h file as requested, thanks